using_openshift/builds.adoc

```diff
@@ -136,15 +163,21 @@ The endpoint can accept an optional payload with the following format:
 [#describe-buildconfig]
 *Displaying a BuildConfig's Webhook URLs*
 
-Use the `osc describe buildConfig [replaceable]#<name>#` command to display the Webhook URLs associated with a build configuration. If no Webhook URLs are displayed, it means that no Webhook trigger is defined for that build configuration.
+Use the `osc describe buildConfig [replaceable]#<name>#` command to display the Webhook URLs associated with
+a build configuration. If no Webhook URLs are displayed, it means that no Webhook trigger is defined for that
+build configuration.
 
 === Image Change Triggers
-Image change triggers allow your build to be automatically invoked when a new version of an upstream image is available. For example, if a build is based on top of a RHEL image, then you can trigger that build to run anytime the RHEL image changes. As a result, the application image is always running on the latest RHEL base image.
+Image change triggers allow your build to be automatically invoked when a new version of an upstream image is
+available. For example, if a build is based on top of a RHEL image, then you can trigger that build to run
+anytime the RHEL image changes. As a result, the application image is always running on the latest RHEL base image.
+
+Configuring an image change trigger requires the following actions:
 
-Configuring an image change trigger requires that a few pieces be in place:
+. First, you must define an ImageRepository that points to the upstream image you want to trigger:
+
+====
```

Comment:
Sorry, should have noticed this sooner. We'll need to be careful with numbered lists and make sure that we use `+` to keep continuous lines of text together that are part of the same step. Otherwise, it starts the numeration back at 1 for each step. So basically many of the blank lines in this list (not the ones actually between steps 1 and 2 and 3, etc, though) need to have a `+` in place to keep things together.

See the "If you need to add any text[...]" bit in [our guidelines](https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#lists) and ["Complex list content"](http://asciidoctor.org/docs/asciidoc-writers-guide/#complex-list-content) from AsciiDoctor.